docs: add release notes in our documentation (#1138)
ansys/pyansys-geometry: doc/source/assets.rst

----------------------------------------

Build the latest Geometry service Docker container using the following assets. Instructions

> on how to build the containers are found at :ref:`Docker containers <ref_docker>`.

Reviewer: ```suggestion
information on how to build the container, see :ref:`Docker containers <ref_docker>`.
```
